Multi tenancy guide update images and some text to include Spaces (#830)
OctopusDeploy/docs: docs/deployment-patterns/multi-tenant-deployments/multi-tenant-deployment-guide/designing-a-multi-tenant-upgrade-process.md

## Step 3: Configure another tenant with multiple environments

Now we will configure a tenant to deploy into multiple environments. If you have been working through this guide, you will already have a tenant you can use.

1. Connect a tenant to the project and the **MT Staging** and **MT Production** environments.

## Step 4: Deploy

### Deploying to Individual Tenants

You can deploy to a single tenant, or a handful of tenants in a batch by selecting them manually. This can be an excellent way to get started with multi-tenant deployments, and it's also a good fallback when you need it.

You can also use the Project Overview to deploy to individual tenants by selecting a release and clicking the **Deploy** buttons.

### Deploying to all tenants in an environment

You can deploy to all of the tenants in an environment using the Octopus UI, the Octopus CLI, or any build-server extensions. Octopus will create a deployment for each tenant, ready for that release to be deployed to the project/environment. Octopus will automatically exclude the tenants who are not ready for that release yet - perhaps they haven't met their lifecycle requirements.

> As a general rule, we don't recommend deploying to all tenants when there are hundreds of tenants. This may cause an unnecessary overload on your Octopus instance, and this is when we would recommend deploying using [Tenant Tags](/docs/deployment-patterns/multi-tenant-deployments/tenant-tags.md).

Reviewer: ```suggestion
As a general rule, we don't recommend deploying to all tenants when there are hundreds of tenants. This may cause an unnecessary overload on your Octopus instance. Instead, we recommend deploying using [Tenant Tags](/docs/deployment-patterns/multi-tenant-deployments/tenant-tags.md).
```
